### ISSUE TITLE
Filter not showing on jet engine CPT

### ISSUE THREAD

**Issue by jeffreytanuwidjaja** [amarinediary/CPT-Admin-Taxonomy-Filtering#1]
hello, i’ve installed your plugin, but it seems that it’s not working

I’m using jetengine to create CPT, and i can’t find the select box option for filter

**Comment by amarinediary** [comment]
Hi jeffrey, there might be two reasons:

1) Either you have empty taxonomies for your custom post type, which in that case are hidden by default if empty (no terms inside a taxonomy).

2) Or, you’ve created hidden/secret taxonomies (not supposed to be queryable by anyone). 

I doupt this issue is related to the jetengine plugin. 

« CPT Admin Taxonomy Filtering » isn’t doing anything fancy beside display ALL taxonomies which are suppose to be queryable onto a CPT admin screen.

**Comment by jeffreytanuwidjaja** [comment]
![image](https://user-images.githubusercontent.com/30138345/135241970-2fa6e3d6-a220-489f-91e2-c8e00e4c410f.png)
hello thanks for the reply

1. theres data on the taxonomy as shown on picture
2. the taxonomy are queryable
![image](https://user-images.githubusercontent.com/30138345/135242289-0bd58723-59a6-4c5f-8f98-b1159155e335.png)

**Comment by amarinediary** [comment]
I don't know what to say. There might be some troubleshoot between both plugins, tho nobody beside you reported anything before.

Bear in mind that "CPT Admin Taxonomy Filtering" only works with Custom Posts Types and doesn't apply to default WordPress Post Types (`post`, `page`, `attachment`, `revision`, `nav_menu_item`).

**Comment by jeffreytanuwidjaja** [comment]
yeah i'm using jetengine to create custom post type, but still no luck, have anyone tried this using jetengine?

thanks
